### DISCUSSION TITLE
Can Fluentd be used to collect old data from MongoDB?

### DISCUSSION THREAD

**Discussion by wahyudisg** [fluent/fluentd#4440] [General]
i have been trying to parsing data log using fluentd to mongodb, but fluentd can't collect the old data, fluentd only transport the new data log. it is possible to collect the old log data using fluentd from mongodb? i using mongo_replset and mongo_tail

**Comment by daipom** [comment]
@tambunanprd 
It looks like `in_mongo_tail` has the setting `id_store_file` and it keeps the last record id.
If you set `id_store_file`, `in_mongo_tail` will resume tailing from the last id.
